apps/staking/src/api.ts

```diff
@@ -140,96 +139,135 @@ type AccountHistory = {
 }[];
 
 export const getStakeAccounts = async (
-  _connection: Connection,
-  _wallet: AnchorWallet,
-): Promise<StakeAccount[]> => {
-  await new Promise((resolve) => setTimeout(resolve, MOCK_DELAY));
-  return MOCK_STAKE_ACCOUNTS;
+  connection: Connection,
+  wallet: AnchorWallet,
+): Promise<StakeAccountPositions[]> => {
+  const pythStakingClient = new PythStakingClient({ connection, wallet });
+  return await pythStakingClient.getAllStakeAccountPositions(wallet.publicKey);
```

Comment:
it is completely redundant. it won't have any effect tho since the function will not be called unless the outer promise is awaited.

Reply:
The promise will be called and dispatched even if not awaited so the behavior is no different.  In the past, eslint would ban `return await` because it [created extra microtasks](https://eslint.org/docs/latest/rules/no-return-await) but that's [no longer true](https://v8.dev/blog/fast-async), so it's actually identical functionally to returning the promise.  I just don't like the semantics much of `return await`.